src/analysis/normed_space/basic.lean

```diff
@@ -179,24 +185,40 @@ nnreal.coe_le.2 $ dist_norm_norm_le g h
 
 end nnnorm
 
+/-- A submodule of a normed group is also a normed group, with the restriction of the norm.
+As all instances can be inferred from the submodule `s`, they are put as implicit instead of
```

Comment:
Should such comments go into the docstring, or should they be a "regular" comment?

Reply:
I don't have an opinion on this. I can change if you want.

Reply:
Let's ask the docstring expert. @robertylewis What do you want?

Reply:
I think it's fine in the docstring.